assets/css/main.css

```diff
@@ -1135,7 +1135,7 @@ input, select, textarea {
 			-webkit-transition: -webkit-transform 0.325s ease-in-out, -webkit-filter 0.325s ease-in-out;
 			-ms-transition: -ms-transform 0.325s ease-in-out, -ms-filter 0.325s ease-in-out;
 			transition: transform 0.325s ease-in-out, filter 0.325s ease-in-out;
-			background-image: url("../../images/bg.jpg");
+			background-image: url("../../images/dbdhoju-2953e0d9-f588-4611-8856-a1669bd066b1.gif");
```

Comment:
It's better to rename images you downloaded from the web that are meaningful so you don't get confused when your codebase grows larger. 😄